lib/ViewModels/FeatureInfoPanelSectionViewModel.js

```diff
@@ -92,11 +92,25 @@ FeatureInfoPanelSectionViewModel.prototype.toggleOpen = function() {
     this.terria.currentViewer.notifyRepaintRequired();
 };
 
+function propertyValues(properties, clock) {
+    // Replace '.' and '#' in property keys with _, since Mustache cannot reference keys with these characters.
+    // In future, we will check each property for a getValue function here too; if it exists, use it to get the current value.
+    var badCharacters = new RegExp('[.#]', 'g');
```

Comment:
I think It's a little better to use `/[.#]/g` instead of an explicit `RegExp` object.